packages/vue-components/src/Navbar.vue

```diff
@@ -306,7 +306,6 @@ export default {
         .navbar-default {
             display: block;
             margin-top: 0.3125rem;
-            width: 100%;
```

Comment:
@kaixin-hc I believe this is still needed so that the `default-navbar` will always go to the next line on mobile :)

Tried with 2 items in the `default-navbar` at about 700px:

![image](https://user-images.githubusercontent.com/47470981/155452577-448cf107-47e8-42d2-8869-bd96f50640a8.png)

Reply:
<img width="760" alt="Screenshot 2022-02-28 at 1 21 48 PM" src="https://user-images.githubusercontent.com/59471320/155928469-7387ce84-c434-4497-9d72-66e6a325a317.png">
Good catch! Though I've enclosed a picture for further reference, as I think that it looks quite strange with just two items. After some consideration I think it's still alright, since I expect this would be an edge case and it doesn't seem to have any usability issues. An alternative work around might be triggering the "go to next line" when the screen size is smaller or when the nav-bar default has a certain number of items, but I think that's more trouble than it's worth